src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 I know there are equipment out there that can handle that speed. However, most of it is very expensive and I think irrelevant when it comes to equipment likely to run UMS. 

Check the price of 10 Gb/s NICs (I've never seen 10 GB/s NIC's yet 😉 ) and not to mention 10 Gb/s switches. I'd also like to see the renderer with a 10 Gb/s NIC 🤣